src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -1,77 +1,66 @@
 
     private ThunderBird climbMotorR;
     private ThunderBird climbMotorL;
 
-    private PositionTorqueCurrentFOC setPointR;
-    private PositionTorqueCurrentFOC setPointL;
+    private PositionVoltage setPointControlR = new PositionVoltage(0d);
+    private PositionVoltage setPointControlL = new PositionVoltage(0d);
 
-    private CLIMBER_STATES state = CLIMBER_STATES.STOW;
-    private Swerve drivetrain;
+    private DutyCycleOut manualControl = new DutyCycleOut(0d);
 
-    private boolean hasClimbed = false;
-    private boolean hasStowed = false;
-    private boolean hasGroundedR = false;
-    private boolean hasGroundedL = false;
-
-    public Climber(Swerve drivetrain) {
+    public Climber() {
         // configure climb motors
         climbMotorR = new ThunderBird(CAN.CLIMB_RIGHT, CAN.CANBUS_FD,
                 ClimbConstants.CLIMB_RIGHT_MOTOR_INVERT, ClimbConstants.CLIMB_MOTOR_STATOR_CURRENT_LIMIT, ClimbConstants.CLIMB_MOTOR_BRAKE_MODE);
         climbMotorL = new ThunderBird(CAN.CLIMB_LEFT, CAN.CANBUS_FD,
             ClimbConstants.CLIMB_LEFT_MOTOR_INVERT, ClimbConstants.CLIMB_MOTOR_STATOR_CURRENT_LIMIT, ClimbConstants.CLIMB_MOTOR_BRAKE_MODE);
 
-        climbMotorL.configPIDF(0, ClimbConstants.EXTEND_KP, ClimbConstants.EXTEND_KI, ClimbConstants.EXTEND_KD);
-        climbMotorL.configPIDF(1, ClimbConstants.RETRACT_KP, ClimbConstants.RETRACT_KI, ClimbConstants.RETRACT_KD);
+        climbMotorL.configPIDF(0, ClimbConstants.UNLOADED_KP, ClimbConstants.UNLOADED_KI, ClimbConstants.UNLOADED_KD);
+        climbMotorL.configPIDF(1, ClimbConstants.LOADED_KP, ClimbConstants.LOADED_KI, ClimbConstants.LOADED_KD);
+
+        climbMotorR.configPIDF(0, ClimbConstants.UNLOADED_KP, ClimbConstants.UNLOADED_KI, ClimbConstants.UNLOADED_KD);
+        climbMotorR.configPIDF(1, ClimbConstants.LOADED_KP, ClimbConstants.LOADED_KI, ClimbConstants.LOADED_KD);
+
+        FeedbackConfigs sensorConf = new FeedbackConfigs();
+        SoftwareLimitSwitchConfigs softLimitConf = new SoftwareLimitSwitchConfigs();
+
+        sensorConf.FeedbackSensorSource = FeedbackSensorSourceValue.RotorSensor;
+        sensorConf.SensorToMechanismRatio = ClimbConstants.GEAR_REDUCTION;
 
-        climbMotorR.configPIDF(0, ClimbConstants.EXTEND_KP, ClimbConstants.EXTEND_KI, ClimbConstants.EXTEND_KD);
-        climbMotorR.configPIDF(1, ClimbConstants.RETRACT_KP, ClimbConstants.RETRACT_KI, ClimbConstants.RETRACT_KD);
+        softLimitConf.ForwardSoftLimitEnable = true;
+        softLimitConf.ForwardSoftLimitThreshold = ClimbConstants.MAX_HEIGHT;
+
+        climbMotorL.applyConfig(climbMotorL.getConfig().withFeedback(sensorConf).withSoftwareLimitSwitch(softLimitConf));
+        climbMotorR.applyConfig(climbMotorR.getConfig().withFeedback(sensorConf).withSoftwareLimitSwitch(softLimitConf));
 
         initLogging();
 
-        climbMotorR.applyConfig();
-        climbMotorL.applyConfig();
+        climbMotorL.setPosition(0d);
+        climbMotorR.setPosition(0d);
```

Comment:
why is this needed

Reply:
I'm fairly certain deploys do retain positions like that, but if this becomes a problem we can comment it out.

You commented out some of the shooter commands and cand shots. Everything else LGTM